### ISSUE TITLE
[FEATURE] Built-in Natural Earth profile

### ISSUE THREAD

**Issue by msbarry** [onthegomap/planetiler#9]
It would be cool to include a profile showcasing all of the vector layers available in [Natural Earth](https://www.naturalearthdata.com/)

Some issues I ran into prototyping this:

- [ ] Need to use Natural Earth shapefile instead of sqlite (and deprecate sqlite one since it's redundant) since some data only exists in the shapefile distribution (i.e. bathymetry)
- [ ] Shapefile reader only supports a single .shp file inside a zip, but natural earth is distributed as many .shp files in a single zip file. There's some debug data in there too so need a way to filter to only some .shp files.
- [ ] Shapefile reader needs to support configurable charsets (Natural Earth uses UTF-8)
- [ ] Some named polygons (i.e. [Physical Labels](https://www.naturalearthdata.com/downloads/10m-physical-vectors/10m-physical-labels/)) are useless in MapLibre, they need to be converted to centerlines (#13) or centroid labels to be useful 
- [ ] Visvalingham-Wyatt simplification would produce better results simplifying geometries at lower zoom levels

**Comment by rene-mueller** [comment]
@wipfli can you possibly help with bathymetry for example?

**Comment by rene-mueller** [comment]
@msbarry possibly the following article can help as well: https://documentation.maptiler.com/hc/en-us/articles/9818655484433-How-to-build-an-ocean-vector-bathymetry-map-with-MapTiler

**Comment by msbarry** [comment]
Good point. If all you want is bathymetry this is pretty straightforward.
